lib/structures/src/map/ord/mod.rs

```diff
@@ -0,0 +1,27 @@
+#[cfg(test)]
+mod tests;
+
+use std::collections::{btree_map, BTreeMap};
+
+#[macro_export]
+macro_rules! ordmap {
+    () => (::structures::map::ord::Map::new());
+
+    // trailing comma case
+    ($($key:expr => $value:expr,)+) => (ordmap!($($key => $value),+));
+
+    ($($key:expr => $value:expr),*) => {
+        {
+            let mut _map = ::structures::map::ord::Map::new();
+            $(
+                let _ = _map.insert($key.into(), $value.into());
+            )*
+            _map
+        }
+    };
+}
+pub type Map<K, V> = BTreeMap<K, V>;
```

Comment:
Ditto my question about standard naming of this type, to prevent confusion when importing `Map`.

Reply:
Yeah, I was wondering why map types appeared in a PR about string types…